changelogs/fragments/2103_better_vcls_missing_datastore_error.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - vmware_cluster_vcls - Fail module with descriptive error if an added datastore does not exist in vcenter
```

```suggestion
  - vmware_cluster_vcls - Fail module with descriptive error if an added datastore does not exist in vcenter
    (https://github.com/ansible-collections/community.vmware/pull/2103).
```

We should add an issue and / or PR to the changelog. This helps people to understand what has been changed an why.